Source/DafnyLanguageServer.Test/DafnyLanguageServer.Test.csproj

```diff
@@ -46,6 +46,18 @@
     <None Update="Lookup\TestFiles\find-refs-b.dfy">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
+    <None Update="Lookup\TestFiles\defines-foo.dfy">
```

Comment:
My preference is for the test files to be inline in the tests. This is similar to what Roslyn's (C# compiler) tests look like. I don't have a good argument for one or the other though.